_posts/2017-09-05-planning-for-7.0.md

```diff
@@ -0,0 +1,498 @@
+
+Our two goals are hand and hand: help developers transform new syntax into backwards compatible code (abstract browser support away), and be a bridge to help TC39 get feedback on new ECMAScript proposals and for the community to have a say in the future of the language.
+
+Thus, I think it's an understatement to say that Babel is an vital part of the JavaScript community and needs it's support. (The only talks I've tried to give are about this point: [JSConf EU](https://github.com/hzoo/maintaining-an-oss-project), [React Rally](https://github.com/hzoo/so-how-does-babel-even-work), [TC39](https://github.com/hzoo/role-of-babel-in-js)). I'll re-iterate a point I made recently: "What happens when Babel dies"? What happens when the current group of people interested in this project get bored/burned out/move on to other things? (What if it's already happened a lot). What are we going to do about it?
+
+## [#4315](https://github.com/babel/babel/issues/4315) Drop support for unmaintained Node versions: 0.10, 0.12, 5
+
+Progress in OSS projects often comes at the cost of upgrading for its users. Because of this, we've always been hesitant in making the choice to introduce a major version bump/breaking changes. But by dropping unsupported versions of Node, we can not only make a number of improvements to the codebase, but also upgrade dependencies and tools (ESLint, Yarn, Jest, Lerna, etc).
+
+## 👓 Proposal Updates/Spec Compliancy
+
+> AKA the only things most of you care about 😅
+
+### Philosophy
+
+We've created a new repo: [babel/proposals](https://github.com/babel/proposals) to track our progress on the various [TC39 Proposals](https://github.com/tc39/proposals) and meetings.
+
+I also added a section about when we accept PRs and new proposals: https://github.com/babel/proposals#when-does-babel-implement-new-features. Our basic thinking is that we will start accepting PRs for anything a TC39 champion is going to present (Stage 0). And we will update (with the help of yall) them when the spec changes.
+
+And naturally, we will take the opportunity to be as spec compliant as possible (within reasonable speed, etc) as the default behavior. This means if you need a faster/smaller build, you should use the `loose` option which will purposely disregard certain spec changes like runtime checks and other edge cases. The reason why it is opt-in is because we expect you should know what you are doing, while others should be able to seamlessly upgrade `babel-preset-env` to use the native version of each syntax or stop using Babel entirely and have no issues.
+
+### Stage 3: Class Properties (from Stage 2)
+
+> `babel-plugin-transform-class-properties`: the default behavior is now what was previously the "spec" option, which uses `Object.defineProperty` instead of simple assignment.
```

Comment:
perhaps worth mentioning the impact this has on the current `transform-decorators` here?

Reply:
Sure ya +1